server/controllers/auth.js

```diff
@@ -58,12 +59,7 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   }
 });
 
-// @route POST /auth/login
-// @desc Login user
-// @access Public
-exports.loginUser = asyncHandler(async (req, res, next) => {
-  const { email, password } = req.body;
-
+const loginHelper = async (res, email, password) => {
```

Comment:
Good job on making a helper method, don't quite like it being in here though. Can we move it to a helpers folder in the server folder?

Reply:
moved this to helpers > loginHelper.js in latest version. thanks!